MediaBrowser.Api/Subtitles/SubtitleService.cs

```diff
@@ -298,5 +316,105 @@ await _subtitleManager.DownloadSubtitles(video, request.SubtitleId, Cancellation
                 }
             });
         }
+
+        public object Get(GetFallbackFontList request)
+        {
+            IEnumerable<FileSystemMetadata> fontFiles = Enumerable.Empty<FileSystemMetadata>();
+
+            var encodingOptions = EncodingConfigurationExtensions.GetEncodingOptions(_serverConfigurationManager);
+            var fallbackFontPath = encodingOptions.FallbackFontPath;
+
+            if (!string.IsNullOrEmpty(fallbackFontPath))
+            {
+                try
+                {
+                    fontFiles = _fileSystem.GetFiles(fallbackFontPath, new[] { ".woff", ".woff2", ".ttf", ".otf" }, false, false);
+
+                    var result = fontFiles.Select(i => new FontFile
+                    {
+                        Name = i.Name,
+                        Size = i.Length,
+                        DateCreated = _fileSystem.GetCreationTimeUtc(i),
+                        DateModified = _fileSystem.GetLastWriteTimeUtc(i)
+                    }).OrderBy(i => i.Size)
+                        .ThenBy(i => i.Name)
+                        .ThenByDescending(i => i.DateModified)
+                        .ThenByDescending(i => i.DateCreated)
+                        .ToArray();
+
+                    // max total size 20M
+                    var maxSize = 20971520;
```

Comment:
Why does the list of fonts have a max size?

Reply:
Since only admins can add fallback fonts I'm not sure we need any size limits.

Reply:
We don't want too large font files to block video loading. But users are likely to use fonts that are not suitable for the web. The upper limit of 20M in total is far enough for subtitle rendering, and it is almost as big as all fonts included in jellyfin-web.

There is also a related PR here, aimed at reducing bandwidth usage in fonts.
https://github.com/jellyfin/jellyfin-web/pull/1087

Reply:
> Since only admins can add fallback fonts I'm not sure we need any size limits.

You never know how the end user will use your program, even if he is an administrator.
It is hard to imagine what will happen if you point the path to `C:\Windows\Fonts`